packages/core/package.json

```diff
@@ -40,5 +40,11 @@
         "dist/**/*",
         "package.json",
         "README.md"
-    ]
+    ],
+    "bugs": { "url" : "https://github.com/doug-wade/tybalt/issues" },
```

Comment:
I think this field is meant to be a link to the currently open bugs for this node module, so I think we can link to the tag we maintain for this package: https://github.com/doug-wade/tybalt/issues?q=is%3Aissue+is%3Aopen+label%3Acore